portal/templates/gil/base.html

```diff
@@ -461,19 +461,19 @@ <h4 class="box-modal__title">Session Timed Out</h4><p class="box-modal__copy">Yo
                       function(data) {
                         if (data) {
                           if (data.name) {
-                            $("#access_code_info").html("Thank you, your access code '" + $("#shortcut_alias").val() + "' indicates you are located at the " + data.name + ". Proceeding to registration . . .");
+                            $("#access_code_info").html("{%trans%}Thank you, your access code '" + $("#shortcut_alias").val() + "' indicates you are located at the " + data.name + ". Proceeding to registration . . .{%endtrans%}");
```

Comment:
Ivan, I tried this, but it seems to literally translate the pass-in characters. e.g. (I see $("shortcut_alias").val() and "data.name" translated literally, instead of the values they represented).  so I put the old changes back and worked.   Let me know if you see problem with my other changes.